### ISSUE TITLE
vlan assignment doesn't work when configured SSD's on an MX

### ISSUE THREAD

**Issue by richardjohnallsopp** [CiscoDevNet/ansible-meraki#193]
When I run the playbook to configure SSID's on the MX, everything works fine bar the vlan assignment, which seems to be assigned  to each SSID by whatever is the lowest SVI vlan ID number.
```
    # create the guest wireless ssid
    - name: enable guest ssid
      meraki_ssid:
        auth_key: "{{   auth_key  }}"
        state: present
        org_name: "{{  org_name  }}"
        net_name: "{{  network_name  }}"
        name: "Wireless-Guest"
        vlan_id: 34
        auth_mode: "psk"
        psk: "password"
        encryption_mode: "wpa"
        enabled: true
      delegate_to: localhost
      tags:
        - "wireless"
    # create the secure wireless ssid
    - name: enable secure ssid
      meraki_ssid:
        auth_key: "{{   auth_key  }}"
        state: present
        org_name: "{{  org_name  }}"
        net_name: "{{  network_name  }}"
        name: "Wireless-Secure"
        vlan_id: 100
        auth_mode: "psk"
        psk: "password"
        encryption_mode: "wpa"        
        enabled: true
      delegate_to: localhost
      tags:
        - "wireless"
``` 
![image](https://user-images.githubusercontent.com/50882889/94184206-77b5a380-fe9b-11ea-8efd-b10e03b525f0.png)

Is there a way to set the vlan that isn't `vlan_id`?

**Comment by kbreit** [comment]
@richardjohnallsopp Thank you for this bug report. The `meraki_ssid` module only supports MR networks and not MX appliances. The module name and documentation aren't obvious about this so in 2.0, I renamed `meraki_ssid` to `meraki_mr_ssid`. I'll need to create a new module for `meraki_mx_ssid`. If I do, are you able to upgrade to version 2.0 of these modules?

**Comment by richardjohnallsopp** [comment]
Sure thing man, happy to help you. It's interesting that they do work for the most part. Just let me know when you're ready for me to upgrade.

**Comment by richardjohnallsopp** [comment]
I've updated the collections to 2.0 :-)

**Comment by kbreit** [comment]
I talked to the Meraki developers and it sounds like their new API v1 (which is what Ansible Meraki 2.0 uses) no longer supports MX SSIDs. I'll keep an eye out for new endpoints for the MX SSIDs and will let you know when that's available.

I will keep this issue open to track that.

**Comment by richardjohnallsopp** [comment]
Hey thanks for the update man, this is a shame as for the most part it actually works, hopefully they will add it back in later on. Seems strange to remove valid functionality from the API.

**Comment by kbreit** [comment]
I feel like there were some challenges with MX and MR support at the same time with the same endpoint. Either way, their suggestion is to submit a Wishlist from the Meraki Dashboard as that helps them triage what new endpoints should be created.

**Comment by kbreit** [comment]
Setting as `wontfix` as I'm waiting on an endpoint for this from Meraki. I'll leave this open for a little bit to see if they get around to it. Even if I close it, I'm still happy to add the module when Meraki adds the endpoint.
